.stylelintignore

```diff
@@ -1 +1,2 @@
 RepClient/src/global.scss
+AdminClient/src/styles.scss
```

Comment:
Should we ignore whole global files?

Reply:
Yeah coz angular / codebot wont pick auto generated classes like in for loops etc and then we would have to allow for some really stupid rules. So i tend to create those only in there and ignore it all together but its up for discussion